gems/aws-sdk-dynamodb/lib/aws-sdk-dynamodb/client.rb

```diff
@@ -190,7 +190,7 @@ class Client < Seahorse::Client::Base
     #
     #     @see https://www.awsarchitectureblog.com/2015/03/backoff.html
     #
-    #   @option options [Integer] :retry_limit (3)
+    #   @option options [Integer] :retry_limit (10)
```

Comment:
this file should not be included in the diff?

Reply:
Discussed offline.